coredb-operator/src/statefulset.rs

```diff
@@ -96,8 +54,55 @@ fn stateful_set_from_cdb(cdb: &CoreDB) -> StatefulSet {
             },
             template: PodTemplateSpec {
                 spec: Some(PodSpec {
-                    containers: vec![postgres_container],
-                    init_containers: Option::from(vec![postgres_init_container]),
+                    containers: vec![
```

Comment:
moved container definitions back into statefulset https://github.com/CoreDB-io/coredb/pull/40#discussion_r1066116171

Reply:
stuff shared between the container and the init container is still outside to avoid duplicating it (e.g. volume mount)